projects/storefrontlib/src/cms-components/cart/cart-coupon/cart-coupon.module.ts

```diff
@@ -20,14 +20,14 @@ import { FormErrorsModule } from '../../../shared/index';
     IconModule,
     FormErrorsModule,
   ],
-  providers: [
-    provideDefaultConfig(<CmsConfig>{
-      cmsComponents: {
-        CartApplyCouponComponent: {
-          component: CartCouponComponent,
-        },
-      },
-    }),
-  ],
+  // providers: [
```

Comment:
- [ ] remove

Reply:
still valid as there is no sample data ready yet